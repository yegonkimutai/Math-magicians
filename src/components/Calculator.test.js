import renderer from 'react-test-renderer';
import Input from './Calculator';

it('renders', () => {
  const tree = renderer.create(<Input />).toJSON();
  expect(tree).toMatchSnapshot();
});
