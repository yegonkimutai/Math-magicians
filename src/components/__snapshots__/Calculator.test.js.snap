// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="calc-wrapper"
>
  <input
    className="input-wrapper"
    readOnly={true}
    type="number"
    value="0"
  />
  <div
    className="btn-row"
  >
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="button"
      onClick={[Function]}
      style={
        Object {
          "background": "darkorange",
        }
      }
      type="button"
    >
      รท
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="button"
      onClick={[Function]}
      style={
        Object {
          "background": "darkorange",
        }
      }
      type="button"
    >
      x
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="button"
      onClick={[Function]}
      style={
        Object {
          "background": "darkorange",
        }
      }
      type="button"
    >
      -
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="button"
      onClick={[Function]}
      style={
        Object {
          "background": "darkorange",
        }
      }
      type="button"
    >
      +
    </button>
    <div
      className="big-grid"
    >
      <button
        className="button"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
    </div>
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="button"
      onClick={[Function]}
      style={
        Object {
          "background": "darkorange",
        }
      }
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
